src/main/java/com/fabriccommunity/spookytime/world/feature/DeceasedWildCropFeature.java

```diff
@@ -0,0 +1,43 @@
+//
+// Source code recreated from a .class file by IntelliJ IDEA
+// (powered by Fernflower decompiler)
+//
+
+package com.fabriccommunity.spookytime.world.feature;
+
+import com.fabriccommunity.spookytime.registry.SpookyBlocks;
+import com.mojang.datafixers.Dynamic;
+import net.minecraft.block.BlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.world.IWorld;
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+import net.minecraft.world.gen.chunk.ChunkGeneratorConfig;
+import net.minecraft.world.gen.feature.DefaultFeatureConfig;
+import net.minecraft.world.gen.feature.Feature;
+
+import java.util.Random;
+import java.util.function.Function;
+
+public class DeceasedWildCropFeature extends Feature<DefaultFeatureConfig> {
+    protected final BlockState crop;
+
+    public DeceasedWildCropFeature(Function<Dynamic<?>, ? extends DefaultFeatureConfig> function_1, BlockState blockState_1) {
```

Comment:
Remove _1 and similar from all variable names

Reply:
Alright.